eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -41,15 +41,11 @@ jobs:
 
     # Linux musl x64
     - ${{ if eq(parameters.platform, 'Linux_musl_x64') }}:
-      - ${{ if eq(parameters.jobParameters.isFullMatrix, false) }}:
```

Comment:
can we test on 3.13 on fullMatrix == true? Meaning on rolling CI?

Reply:
Yup, didn't mean to delete on rolling ones 🤣